apps/ledger-live-mobile/e2e/page/discover/discover.page.ts

```diff
@@ -0,0 +1,26 @@
+import {
+  getElementById,
+  getElementByText,
+  openDeeplink,
+  waitForElementByText,
+} from "../../helpers";
+import { expect } from "detox";
+
+const baseLink = "discover/";
+
+export default class DiscoverPage {
+  waitForSelectCrypto = () => waitForElementByText("Select crypto");
+  discoverPageHeader = () => getElementById("discover-banner");
+
+  async openViaDeeplink(discoverApps = "") {
+    await openDeeplink(baseLink + discoverApps);
+  }
+
+  async expectUrl(url: string) {
+    await expect(getElementByText(url)).toBeVisible();
+  }
+
+  async expectDiscoverPage() {
+    await expect(this.discoverPageHeader()).toBeVisible();
```

Comment:
Same here, I don't think that we need to use await, then expectDiscoverPage could be synchronous

Reply:
Same, [toBeVisible](https://wix.github.io/Detox/docs/next/api/expect#tobevisible) from Detox's asynchronous expect.